src/rideables/StackTest.cpp

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#include <iostream>
+#include "Stack.hpp"
+
+using namespace std;
+
+int main(int argc, char const *argv[])
+{
```

Comment:
You can write a unit test like this to check the correctness of some particular module, but try to do it with multiple threads. Alternatively, you can write a test like [QueueChurnTest.cpp](https://github.com/urcs-sync/Montage/blob/master/src/tests/QueueChurnTest.hpp), which will handle multithreading part by its own. Ignore anything inside `#ifdef PRONTO`.